scorep/instrumenters/scorep_trace.py

```diff
@@ -98,7 +102,11 @@ def localtrace_trace(self, frame, why, arg):
             code = frame.f_code
             modulename = frame.f_globals.get('__name__', None)
             if modulename is None:
-                modulename = "None"
+                # this is a NUMPY special situation, see NEP-18, and Score-P Issue issues #63
```

Comment:
Make a function for this to reduce code duplication. You've got that code in 4 different places.

Reply:
See #66 . There are some more changes needed to mage this consistent.